Sources/Core/GTMSessionFetcher.m

```diff
@@ -139,6 +139,7 @@ @interface GTMSessionFetcher ()
 
 @property(atomic, strong, readwrite, nullable) NSData *downloadedData;
 @property(atomic, strong, readwrite, nullable) NSData *downloadResumeData;
+@property(atomic, readwrite) BOOL cancelAndCallback;
```

Comment:
Note: all the fetcher's `atomic` properties are explicitly implemented and do `@synchronized(self)` rather than rely on the generated "atomic" getter/setter (which almost never does what devs really want).